fetch.bs

```diff
@@ -3061,6 +3251,31 @@ in addition to <a>HTTP fetch</a> above.
   nullity has already been checked. The <a lt=extract for=BodyInit>extracting</a> operation cannot
   throw as it was called for the same <a for=body>source</a> before.
 
+ <li><p>Let <var>useReferredTokenBinding</var> be the result of
+   <a for=header>parsing</a> `<code>Include-Referred-Token-Binding-ID</code>`
+   in <var>actualResponse</var>'s <a for=response>header list</a>.
```

Comment:
The Include-Referred-Token-Binding-ID header is meaningful only if the connection that it was received on had Token Binding enabled. It should not matter if the response comes from a Service Worker.